qemu/tests/scsi_mq_check.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+
+
+def get_mq_from_guest(session, params, cmd, pattern):
+    """
+    Get multi queue number from guest.
+
+    :param session: VM session
+    :param params: Dictionary with the test parameters
+    :param cmd: command to get queue info
+    :param pattern: Get the queue number from queue info
+    """
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        logging.error("cmd running failed with output %s" % output)
+    if params.get("os_type") == 'linux':
+        queues = len(re.findall(pattern, output, re.I | re.M))
+    if params.get("os_type") == "windows":
+        queues = re.search(pattern, output, re.I | re.M).group(1)
```

Comment:
if it's better to write like this:
if params.get("os_type") == 'linux':
    xxxxx
elif params.get("os_type") == 'windows':
    xxxx
else:
    test.error(not support os type)

Reply:
Hi xiagao, for this problem, wyu confirmed with xuhan before, no need to consider others os_type, except 'linux' and "windows" now~